clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -9311,6 +9311,9 @@ def note_constinit_missing_here : Note<
 def err_dimension_expr_not_constant_integer : Error<
   "dimension expression does not evaluate to a constant unsigned int">;
 
+def err_function_parameter_limit_exceeded : Error<
+  "too many function parameters; subsequent parameters are skipped">;
```

Comment:
```suggestion
  "too many function parameters; subsequent parameters will be ignored">, DefaultFatal;
```

Reply:
I changed the wording, but I don't agree with `DefaultFatal`. We use it very sparingly (only 51 result across all of our diagnostics), and now we're successfully recovering.
CC @AaronBallman

Reply:
If we can successfully recover, then it's better to drop the `DefaultFatal` so users can get additional follow-on diagnostics. `DefaultFatal` is more for "there's no decent recovery and continuing would be a bad idea"